app/assets/javascripts/discourse/tests/acceptance/do-not-disturb-test.js

```diff
@@ -48,6 +48,33 @@ acceptance("Do not disturb", function (needs) {
     );
   });
 
+  test("Can be invoked via keyboard", async function (assert) {
+    updateCurrentUser({ do_not_disturb_until: null });
+
+    await visit("/");
+    await click(".header-dropdown-toggle.current-user");
+    await click(".menu-links-row .user-preferences-link");
+
+    await click(".do-not-disturb");
+
+    assert.ok(exists(".do-not-disturb-modal"), "modal to choose time appears");
+
+    let tiles = queryAll(".do-not-disturb-tile");
+    assert.ok(tiles.length === 4, "There are 4 duration choices");
+
+    await triggerKeyEvent(".do-not-disturb-tile:nth-child(1)", "keydown", 13);
+
+    assert.ok(
+      query(".do-not-disturb-modal").style.display === "none",
+      "modal is hidden"
+    );
+
+    assert.ok(
+      exists(".header-dropdown-toggle .do-not-disturb-background"),
+      "moon icon is present in header"
```

Comment:
I think the word is an internal implementation detail and the query above does not reflect that as well. Should the query be more specific in this case?